pkg/summarizer/flakiness_test.go

```diff
@@ -482,14 +483,18 @@ func TestParseGrid(t *testing.T) {
 		},
 	}
 
+	metricsSort := func(x *common.GridMetrics, y *common.GridMetrics) bool {
+		return x.Name < y.Name
+	}
+
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
 			actualMetrics, actualFS := parseGrid(tc.grid, tc.startTime, tc.endTime)
-			if !reflect.DeepEqual(actualMetrics, tc.expectedMetrics) {
-				t.Errorf("Metrics disagree:\nactual %+v \n!= \nexpected %+v", actualMetrics[0], tc.expectedMetrics[0])
+			if diff := cmp.Diff(tc.expectedMetrics, actualMetrics, cmpopts.SortSlices(metricsSort)); diff != "" {
+				t.Errorf("Metrics disagree (-want +got):\n%s", diff)
 			}
-			if !reflect.DeepEqual(actualFS, tc.expectedFilteredStatus) {
-				t.Errorf("Status disagree:\nactual %+v \n!= \nexpected %+v", actualFS, tc.expectedFilteredStatus)
+			if diff := cmp.Diff(tc.expectedFilteredStatus, actualFS); diff != "" {
+				t.Errorf("Status disagree (-want +got):\na%s", diff)
```

Comment:
I think there's an extraneous 'a' in there? (`\na` should be `\n`?)

Reply:
Woops, you're right. Done.